### ISSUE TITLE
example-forum emails still using pre-1.12 API

### ISSUE THREAD

**Issue by jimrandomh** [VulcanJS/Vulcan-Starter#97]
The Vulcan 1.12 migration guide (https://medium.com/@sachagreif/vulcan-js-1-12-a-better-graphql-api-for-crud-operations-fae82e03a7e9) describes some things that need to be done to make email sending work in 1.12. However, it looks like `Vulcan-Starter/packages/example-forum/lib/modules/notifications/emails.js`  hasn't had the migration applied. I was hoping to look at that file to see an example of how the new email API works.

**Comment by SachaG** [comment]
You're right, I don't have much time to work on that right now so hopefully someone can submit a PR. Here's an example from http://zenshome.jp using the new API: 

https://gist.github.com/SachaG/1f1c50408aaa2ca4c43d3b27b34e2007

**Comment by lc3t35** [comment]
+1, please update documentation give some examples.
